xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -85,21 +95,90 @@ final class DatabaseService {
         }
     }
 
+    /// Deletes and rebuilds the database by applying migrations to a new
+    /// database.
+    func rebuild() throws -> Int {
```

Comment:
Rebuild can be used to delete and reconstruct database, recovering from a broken state.